configs/efficientdet/README.md

```diff
@@ -0,0 +1,15 @@
+# EfficientDet: Scalable and Efficient Object Detection
+
+## Introduction
+EfficientDet family of models based on [EfficientDet paper](arxiv.org/pdf/1911.09070.pdf).
+
+Main features of EfficientDet models are EfficientNet backbone and BiFPN neck.
+
+## Results and Models
+|     Backbone    |    Head    |  Style  | Lr schd | Mem (GB) | Inf time (fps) | box AP | Download |
+| :-------------: | :--------: | :-----: | :-----: | :------: | :------------: | :----: | :------: |
+| efficientnet_b0 | retinahead | pytorch |   1x    |   22.0   | 33.3           |  30.2  | [model](https://storage.openvinotoolkit.org/repositories/mmdetection/models/efficientdet/retinanet_effd0_bifpn_1x_coco/epoch_300.pth) |
```

Comment:
* `1x` schedule is 12 epoches. According to the config file, you run training for 300 epoches. If so, you should put `300e` here.
* "For fair comparison with other codebases, we report the GPU memory as the maximum value of torch.cuda.max_memory_allocated() for all 8 GPUs. Note that this value is usually less than what nvidia-smi shows". Please confirm that this is the way you have obtained the memory consumption.
* Add link to the config file, as in other models' readme files.
* What GPUs have you used for training/benchmarking?

Reply:
- Fixed lr schd and config link, can be seen on newest README
- mmdetection outputs the amount of memory (in MB) in its logs ([code link](https://github.com/open-mmlab/mmcv/blob/d5241b70871c0664a1584989b58d7c507c240278/mmcv/runner/hooks/logger/text.py#L163)). The value in the logs is 22037 MB, or 22.0 GB
- I used 4 RTX 6000 cards to train the model. I provided that as a note on the README